src/components/AxisControls/index.js

```diff
@@ -1,5 +1,6 @@
 'use strict';
 
+var $ = require('jquery');
```

Comment:
I should probably unify usages of var vs. const across the app - will add it to todo list.